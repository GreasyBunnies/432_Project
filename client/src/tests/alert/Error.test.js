import React from 'react';
import ReactDOM from 'react-dom';
import Error from '../../components/alert/Error.js';

import 'jest-localstorage-mock';
import renderer from 'react-test-renderer';

// Error
it('renders Error without crashing', () => {
	const div = document.createElement('div');

	ReactDOM.render(<Error />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('displays the correct message', () => {
	const component = renderer.create(
		<Error message="This is a test"></Error>,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});