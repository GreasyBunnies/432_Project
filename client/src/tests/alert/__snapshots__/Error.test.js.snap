// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Error alert with message Displays the correct message 1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    An Error Has Occurred
  </div>
  <p>
    This is a test
  </p>
</div>
`;
